client/lobbies/socket-handlers.js

```diff
@@ -230,6 +230,7 @@ const eventToAction = {
   },
 
   cancelLoading: (name, event) => dispatch => {
+    clearCountdownTimer()
```

Comment:
fadeAtmosphere seems like the more correct call, the countdown would never actually be running here